src/components/Radio/Radio.tsx

```diff
@@ -1,15 +1,89 @@
-interface RadioOption {
-  variable: string;
-  value: number;
-}
-
-interface RadioProps {
-  options: RadioOption;
-  onChange: () => void;
-}
-
-const Radio = (_: RadioProps) => {
-  return <></>;
+import React from 'react';
+import styles from './radio.module.css';
+import utils from '../../styles/modules/utils.module.css';
+
+const ColorOption = ({
+  color,
+  isFirstOption,
+}: {
+  color: string;
+  isFirstOption: boolean;
+}) => (
+  <>
+    <input
+      type="radio"
+      name="color"
+      id={color}
+      className={styles.colorRadio}
+      defaultChecked={isFirstOption}
+    />
+    <label htmlFor={color} className={styles.colorOption}>
```

Comment:
This is wrong since label don't have any content, please recheck the label tag

Reply:
Fixed